data/apple/245efb70-84f4-11e6-9ac7-f5fcfad695e3.json

```diff
@@ -0,0 +1 @@
+{"id":"245efb70-84f4-11e6-9ac7-f5fcfad695e3","type":"example","color":"colorexample","size":"examplesize"}
```

Comment:
@loganlsr no need to push up your JSON data files - add these to your `gitignore` to avoid overpopulating the repo